### DISCUSSION TITLE
Important to compute capsize results separately for each font size?

### DISCUSSION THREAD

**Discussion by dabrahams** [seek-oss/capsize#65] [Q&A] [answered]
The trim amounts being specified in `em`s identically for every `font-size`, and the `lineHeight` being passed through capsize without modification (except for rounding to 4 decimals), it seems like the rounding is the only thing that is producing `font-size`-specific results, and I'm wondering what difference that makes and whether there's some reason I'm missing that dictates I should do the capsize computations for every font size I use?

Thanks,
Dave

**Comment by michaeltaranto** [comment]
The trim has to be performed for each unique combination of font size and line height, as the amount to be trimmed above/below will vary based on the requested sizes. The values are changing for me, do you have an example where they are not changing for you?

**Comment by dabrahams** [comment, marked as answer]
Yeah, I guess what I'm seeing is that for any ratio of `lineGap` to `fontSize`, I get the same trim values.  Therefore when `lineGap` is zero, the trim values are the same at any font size.  And if `lineGap` were actually specified in `em`, then I think the trim values would be completely `lineGap`-independent.

**Reply by michaeltaranto** [reply]
If your ratios between line height and font size are exactly the same across your entire hierarchy and using the same font, then yes the computed values will be the same.
